docs/development/decision-records/2024-08-28-Release-Workflow-Refactor/README.md

```diff
@@ -0,0 +1,82 @@
+# Refactor of the Release Workflow
+
+## Decision
+
+Refactor the existing workflows to accommodate the release of bugfix versions as described in [this](../2024-07-18-Improvements-versioning-scheme/README.md) decision record:
+- Remove the releases and release/X.Y.Z branches in favor of using just the main branch and tags.
```

Comment:
this will not work for subsequent bugfixes: imagine we have a `0.8.1` and then need to create a `0.8.2`. What would `0.8.2` branch off of?

Reply:
The 0.8.1 tag? Sorry, i'm not understanding the issue.

Reply:
yeah, sorry, took a mental cul-de-sac :/